modules/hcp-virt-add-node.adoc

```diff
@@ -44,7 +44,7 @@ example                   example         5               5               False
 example-extra-cpu         example         2                               False         False                  True              True             Minimum availability requires 2 replicas, current 0 available
 ----
 +
-Replace `4.x.0` with the supported {product-title} version that you want to use.
+Replace `<4.x.0>` with the supported {product-title} version that you want to use.
```

Comment:
On line 43, be sure to place `4.x.0` in angle brackets to match the annotation text.